cmd/root.go

```diff
@@ -527,6 +527,9 @@ CPU may be throttled and a background refresh cannot run reliably
 against all specified instances. If an instance is unreachable, the Proxy exits with a failure
 status code.`)
 
+	localFlags.BoolVar(&c.conf.IgnoreFailedConnections, "ignore-failed-connections", false,
```

Comment:
Let's make this skip-failed-instance-config, and update https://github.com/GoogleCloudPlatform/cloud-sql-proxy/blob/main/migration-guide.md to mention this new flag.